validator/webui/index.html

```diff
@@ -389,6 +391,7 @@
 
           // The status bar below the editor.
           var statusBar = this.$.statusBar;
+          statusBar.status = 'PASS';  // Start out valid.
```

Comment:
It's a good point but I think on balance it's best as is. I don't want to require the user to clear out the URL, and this content is different than minimum_valid_amp.html and loads faster this way. If this becomes invalid we'll have to update it, but it's easy compared to worrying about all those AMP pages that aren't checked into our codebase. :-)